README.md

```diff
@@ -1 +1,131 @@
+  - [Setup](#setup)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Bookstore
+
+The Bookstore is a website that allows you to:
+ - Display a list of books
+ - Add a book
+ - Remove a selected book
+
+## Built With
+
+- React.js
+- Redux Toolkit
+- VSCode and GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Live Demo
+
+[Live Demo Link](Not available)
+
+<!-- Features -->
+
+### Key Features
+
+- **[Git Flow]**
+- **[Best practices for JavaScript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+
+Clone this repository to get started with the project, make sure to configure linters by following the instructions in the [Linters README file](https://github.com/microverseinc/linters-config/blob/master/README.md)
+
+### Setup
+
+- Clone the folder to your local machine.
+- Open a local copy of the project in your code editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+- To run the project, execute the following command:
+
+    - <code>npm start</code> in your terminal.
+
+- To run test functions, execute the following command:
+
+    - <code>npm test</code> in your terminal.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+- After pushing the changes on github, merge your pull request and let github action process changes and fix errors, if any.
+  - For stylelint errors run:<code>npx stylelint "**/*.{css,scss}" --fix</code> for automatic fix.
+  - For ESLINT errors run:<code>npx eslint "**/*.{js,jsx}" --fix</code> for automatic fix.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors
+
+👤 **Ashmal Zahra**
+
+- GitHub: [@githubhandle](https://github.com/ashmalzahra)
+- Twitter: [@twitterhandle](https://twitter.com/AshmalZahraa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ashmal-zahra-35bb09242/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add reducers and actions**
+- [ ] **Use Redux in React components**
+- [ ] **Connect to API.**
+- [ ] **Styling**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ashmalzahra/math-magicians/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] kindly create a file in your root directory called `MIT.md`  copy the MIT licence into it and link the file(MIT.md) in your README file